src/main/java/core/basesyntax/shop/model/Fruit.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.shop.model;
+
+import java.util.Objects;
+
+public class Fruit {
```

Comment:
Class must be immutable as Ksenia told.

Reply:
since it has immutable string field and i've overrode equals and hashcode i think it's ok

Reply:
ur class isnt final [link](https://mate.academy/learn/java-core/java-immutable/theory?section=theory), but its not necessary